tasks/testwasher.py

```diff
@@ -174,13 +175,21 @@ def get_tests_family(self, test_name_list):
 
 
 @task
-def generate_flake_finder_pipeline(ctx, n=3):
+def generate_flake_finder_pipeline(ctx, n=3, generate_config=False):
```

Comment:
I think we should, so that we don't need to use a gitlab secret in the test washer related jobs. That's normally what I did [there](https://github.com/DataDog/datadog-agent/pull/29950/files#diff-08bac3144036ba1baff79ea26d0091357c83d0abfe2fd21d5aed2edcaf043c73R482)